src/views/admin/PersonDetailsView.vue

```diff
@@ -1380,6 +1394,13 @@
                   @update:canCommit="canCommit = $event"
                   @fieldReset="handleFieldReset"
                 />
+                <KopersInput
+                  v-if="!hasKopersRolle && isKopersRolle(selectedRolle) && selectedOrganisation"
```

Comment:
`HasKopersRolle ` : Das wird bei SPSH-811 benutzt um die vorhandenen Kontexte auszuwerten. Wenn mindestens 1 Rolle Koperspflicht ist dann zeigen wir den Text "Fehlt" in Rot. 

Wenn wir aber das hier so nutzen heisst das: KopersNr Inputfield wird nie gezeigt im Falle, wenn wir eine Person mit einer bestehenden Rolle, dass Koperspflichtig ist... Das heisst dann auch: Ich kann die KopersNr gar nicht nachreichen. 

Ich würde das hier also entfernen.